packages/manager/test/SliceMachineManager-slices-createSlice.test.ts

```diff
@@ -12,6 +12,7 @@ it("calls plugins' `slice:create` hook", async (ctx) => {
 	const adapter = createTestPlugin({
 		setup: ({ hook }) => {
 			hook("slice:create", hookHandler);
+			hook("slice:asset:update", hookHandler);
```

Comment:
Since we're spying on this hook later, using the same spy here could have unexpected side effects later on, instead you could just use `vi.fn()` inline directly here.